policies/recipes/steps.go

```diff
@@ -131,27 +132,100 @@ func StepFileExec(step *osconfigpb.SoftwareRecipe_Step_FileExec, artifacts map[s
 
 // StepScriptRun builds the command for a ScriptRun step
 func StepScriptRun(step *osconfigpb.SoftwareRecipe_Step_ScriptRun, artifacts map[string]string, runEnvs []string, stepDir string) error {
+	if runtime.GOOS == "windows" {
```

Comment:
I think this would look cleaner if you had this check wihin each case that it mattered.

Reply:
Done.